ios/RNSentry.mm

```diff
@@ -195,6 +199,86 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
     resolve(modulesString);
 }
 
+RCT_EXPORT_METHOD(fetchNativePackageName:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject)
+{
+    NSString *packageName = [[NSBundle mainBundle] executablePath];
+    resolve(packageName);
+}
+
+- (NSDictionary*) fetchNativeStackFramesBy: (NSArray<NSNumber*>*)instructionsAddr
+                               symbolicate: (SymbolicateCallbackType) symbolicate
+{
+  BOOL shouldSymbolicateLocally = [SentrySDK.options debug];
```

Comment:
H: Does the Cocoa SDK do that too based on the `debug` flag? I believe that this should be based on the build type and not on the sentry options debug flag.

Reply:
Cocoa does it also based on the `debug` flag from clients' options.
https://github.com/getsentry/sentry-cocoa/pull/3079

Reply:
Ok, lets keep as it is then